target/linux/ath79/image/generic-ubnt.mk

```diff
@@ -80,49 +81,52 @@ endef
 
 define Device/ubnt_airrouter
   $(Device/ubnt-xm)
-  DEVICE_TITLE := Ubiquiti AirRouter
+  DEVICE_MODEL := AirRouter
   SUPPORTED_DEVICES += airrouter
 endef
 TARGET_DEVICES += ubnt_airrouter
 
 define Device/ubnt_bullet-m
   $(Device/ubnt-xm)
-  DEVICE_TITLE := Ubiquiti Bullet-M
+  DEVICE_MODEL := Bullet-M
   SUPPORTED_DEVICES += bullet-m
 endef
 TARGET_DEVICES += ubnt_bullet-m
 
 define Device/ubnt_bullet-m-xw
   $(Device/ubnt-xw)
-  DEVICE_TITLE := Ubiquiti Bullet-M (XW)
+  DEVICE_MODEL := Bullet-M
+  DEVICE_VARIANT := XW
```

Comment:
Here, as with the AR300M "NAND", you are removing the brackets around XW. As this is not part of the device name, I would think about using `DEVICE_VARIANT := (XW)` here. However, this is to be considered an option only.